draft-looker-jwt-cwt-status-list.md

```diff
@@ -180,10 +199,12 @@ TODO elaborate on issuers avoiding sequential usage of indices and status lists
 TODO elaborate that a status list only gives information about the maximum number of possible statuses that a list conveys, issuers are recommended to pre-allocate lists, use dead entries that are never assigned or other obfuscation mechanisms
 
 ## Malicious Issuers
-TODO elaborate on issuers generating unique status lists per JWT token that do not offer herd privacy
+TODO elaborate on issuers generating unique status lists per Referenced Token that do not offer herd privacy
```

Comment:
Can we keep TODOs as issues, pls.? It makes it easier to grasp the TODOs for a certain draft.

Reply:
these TODOs are partly resolved in other PRs and I would like to not change them in this PR as this resulsts to even more merge conflicts